gibbon-rts/rts-c/gibbon_rts.c

```diff
@@ -1796,13 +1796,19 @@ void gib_show_usage(char** argv)
     printf("\n");
     printf("This binary was generated by the Gibbon compiler.\n");
     printf("\n");
-    printf("Usage: %s [OPTS] [size] [iters]\n", argv[0]);
+    printf("Usage: %s [OPTIONS...]\n", argv[0]);
 
     printf("\n");
     printf("Options:\n");
-    printf(" --buffer-size <bytes>      Set the buffer size (default %" PRId64 ").\n", gib_global_biginf_init_chunk_size);
-    printf(" --bench-input <path>       Set the input file read for benchmarking. Applies only\n");
-    printf("                            IF the program was *compiled* with --bench-fun. \n");
+    printf(" --biginf-buffer-size <bytes>   Set the buffer size (default %" PRId64 ").\n", gib_global_biginf_init_chunk_size);
+    printf(" --inf-buffer-size <bytes>      Set the buffer size (default %" PRId64 ").\n", gib_global_inf_init_chunk_size);
+    printf(" --bench-input <path>           Set the input file read for benchmarking. Applies only\n");
+    printf("                                IF the program was *compiled* with --bench-fun. \n");
```

Comment:
why `IF` is capitalized?

Reply:
I'm not sure why it was capitalized to begin with. I can make it lower case.